### ISSUE TITLE
[Logging] Implement logging hotswap endpoint with presets support

### ISSUE THREAD

**Issue by silasdavis** [hyperledger/burrow#805]
Burrow produces a large volume of logging with no filters and `ExcludeTrace = false`. This can be a burden when capturing logs across many validators and chains. Often we are temporarily interested in an increased amount of logging - often regarding a particular area, e.g. consensus, p2p, EVM, accounts,... For example when deploying contracts with Bos we may want to up the EVM logging to max in order to debug issues. Similarly if a node has lost connectivity it is useful to ramp up the p2p logging.

Burrow has two core latent features already implemented that can be combined to provide just this functionality.

1. The logger supports concurrent hot swapping of its output logger (which can contain an arbitrary pipeline of filters and transforms): https://github.com/hyperledger/burrow/blob/develop/logging/logger.go#L112-L114

2. There exists a system of logging presets that can be defined to provide a particular verbosity or topic focused logging: https://github.com/hyperledger/burrow/blob/develop/logging/config/presets/instructions.go

What we would like to do is:
- Define a logging service in our new GRPC layer: https://github.com/hyperledger/burrow/blob/develop/rpc/burrow/burrow.proto (for reference see: https://github.com/hyperledger/burrow/blob/develop/rpc/burrow/transaction_server.go)
- Have this endpoint support a `Swap` method that allows you to specify either a preset logging mode from instructions.go or a custom logging config provided as JSON/TOML payload. Either config can be used to hotswap the output logger to a new one.

Once that basic functionality is in place it would be nice to have:
- Logging presets defined in Burrow config to augment hard-coded ones
- The ability to define new presets at runtime (by supplying a name when calling `Swap` with a custom config)

To make this really useful we should think about presets that are operationally useful and try and come up with some good default presets. We can work with the Monax product and operations team who have good experience in production and can provide feedback.

Here is a suggested logging RPC protobuf interfacce:

```protobuf
service Logger {
    rpc Swap (SwapLoggerRequest) returns (SwapLoggerResponse);
}

message SwapLoggerRequest {
    // If provided with LoggingConfig - try to save the provided config as this preset name
    // If provided without LoggingConfig try to load an existing preset with this name
    string PresetName = 1;
    // JSON or TOML logging config
    string LoggingConfig = 2;
}

message SwapLoggerResponse {

}
```

**Comment by silasdavis** [comment]
I think we should just log less
